site/en/getstarted/prerequisite-docker.md

```diff
@@ -30,13 +30,27 @@ Before you install Milvus, check your hardware and software to see if they meet
 | Linux platforms            | <ul><li>Docker 19.03 or later</li><li>Docker Compose 1.25.1 or later</li></ul> | See [Install Docker Engine](https://docs.docker.com/engine/install/) and [Install Docker Compose](https://docs.docker.com/compose/install/) for more information. |
 | Windows with WSL 2 enabled | Docker Desktop                                               | We recommend that you store source code and other data bind-mounted into Linux containers in the Linux file system instead of the Windows file system.<br/>See [Install Docker Desktop on Windows with WSL 2 backend](https://docs.docker.com/desktop/windows/install/#wsl-2-backend) for more information. |
 
-| Software | Version                       |
-| -------- | ----------------------------- |
-| etcd     | 3.5.0                         |
+| Software | Version                       | Note |
+| -------- | ----------------------------- | ---- |
+| etcd     | 3.5.0                         |  See [additional disk requirements](###-additional-disk-requirements). |
```

Comment:
You need to add a pipe at the end of the other two rows below to form a complete table. 
See https://www.markdownguide.org/extended-syntax/#tables

Reply:
Error rectified.